src/Explorer/Controls/NotebookGallery/Cards/GalleryCardComponent.tsx

```diff
@@ -148,6 +149,14 @@ export class GalleryCardComponent extends React.Component<GalleryCardComponentPr
     );
   }
 
+  private renderTruncatedDescription = (): string => {
+    let truncatedDescription = this.props.data.description.substr(0, GalleryCardComponent.cardDescriptionMaxChars);
+    if (truncatedDescription.length === GalleryCardComponent.cardDescriptionMaxChars) {
```

Comment:
May be we should check if description length is more than the substring

Reply:
modified it,